app/admin/views.py

```diff
@@ -247,6 +254,35 @@ def destinations_delete(uuid):
     )
 
 
+def _make_destination_action_email_messages(destination, verb, template_base):
+    messages_to_send = []
+    for carpool in destination.carpools:
+        subject = 'Carpool on {} {}'.format(carpool.leave_time, verb)
+        people = [
+            ride_request.person for ride_request in carpool.ride_requests
+        ]
+        people.append(carpool.driver)
+        messages_to_send.extend([
+            make_email_message(
+                'admin/destinations/email/{}.html'.format(template_base),
+                'admin/destinations/email/{}.txt'.format(template_base),
+                person.email,
+                subject,
+                destination=destination,
+                carpool=carpool,
+                person=person) for person in people
```

Comment:
This syntax is a little bit tricky to spot. Can you do something a little more obvious like:

```python
for person in people:
  messages_to_send.append(make_email_message(...))
```